### PR TITLE
complete angular hit function, haven't gone through testing

### PR DESCRIPTION
**Pull request by nukenukenukelol**
And it's my first time using matlab woohoo

### CODE REVIEW

File: polarCoordinateTraversal.m [hunk 1]
```diff
@@ -194,21 +194,57 @@ function polarCoordinateTraversal(min_bound, max_bound, ray_origin, ray_directio
...
+        is_angular_hit = false;
+    end
+    if zmax(0) < 0 | zmin(1) < 0
+        is_angular_hit = false;
+    end
+    
+    % If we hit the min boundary then we decrement theta, else increment;
+    % assign tmaxtheta
+    if zmin(0) < 0 | zmin(1) < 0 
+        tDeltaTheta = -1;
```
Comment (cgyurgyik) [inline polarCoordinateTraversal.m]: Why +1/-1 for tDeltaTheta? This may be more appropriate for @ak-2485

**Comment by cgyurgyik** [comment]
Also in the future, make sure to assign someone to review your PR so we get the notification

File: polarCoordinateTraversal.m (hunk 1 above)
Comment (ak-2485) [reply polarCoordinateTraversal.m]: tDeltaTheta is analogous to stepX, StepY from Aman. & Woo. In fact, we could call it stepTheta if you prefer.

Comment (cgyurgyik) [reply polarCoordinateTraversal.m]: I think that'd make more sense yes. Can you change that @nukenukenukelol ?

**Pull request merged**